### PR TITLE
Feature/local simc

### PR DESCRIPTION
**Pull request by lutzmor**
### Description of the Change

Added a possibility to use simc localy instead of raidbots. I used the approach with minimal changes to the current behavior of the python scripts. 

The profiles.py keeps unchanged so you create the same profiles for local and raidbots sims. In sim.py i introduced a new argument --local with this set he uses the logic in internal/simc.py the rest of the logic stays the same so only the import of "raidbots" and the api_key got changed.

### Benefits

Well :) we could use simc local if wanted ;)

### CODE REVIEW

File: sim.py [hunk 1]
```diff
@@ -87,6 +141,8 @@ def main():
         '--talents', help='indicate talent build for output.', choices=config["builds"].keys())
     parser.add_argument(
         '--covenant', help='indicate covenant build for output.', choices=config["covenants"])
+    parser.add_argument(
+        '--local', help='indicate covenant build for output.', action='store_true')
```
Comment (seanpeters86) [inline sim.py]: should change help text here

File: sim.py [hunk 2]
```diff
@@ -59,9 +113,9 @@ def run_sims(args, iterations, talent, covenant):
             raidbots(api_key, profile_location,
                      config["simcBuild"], output_location, profile_name_with_dir, iterations)
         elif weight == 0:
-            print("{0} has a weight of 0. Skipping file.".format(output_name))
+            print("-- {0} has a weight of 0. Skipping file.".format(output_name))
```
Comment (seanpeters86) [inline sim.py:115]: what do these dashes do?

**Review by seanpeters86 — CHANGES_REQUESTED** [review]
LGTM, just a few comments.

Also, when running this locally with this method were you able to generate results properly? Looks like it should, just curious.

File: sim.py (hunk 2 above)
Comment (lutzmor) [reply sim.py:115]: This dashes don't do much :), just formatting the output a bit: So it looks like this
Simming 6 out of 18
-- consumables_lm_na_2.json has a weight of 0. Skipping file.

Did this do have a "better" overview in the shell when there are around 100 - 200 lines to see what happens, and where i have problems. I produce similar lines for example when something in the sim itself run's to an error

Simming 1 out of 18
-- consumables_hm_na_1.json has an error. Skipping file.
-- simc: /home/espo/projects/simc/engine/action/sc_action.cpp:93: virtual void {anonymous}::queued_action_execute_event_t::execute(): Assertion `action->cooldown->ready <= sim().current_time()' failed.

So i can see at the end of an sim when i return to the pc quite quickly what happend and what got skipped and so on.

File: sim.py (hunk 1 above)
Comment (lutzmor) [reply sim.py]: Should be fixed, damn copied it over from the covenant and forgot to change it.

**Comment by lutzmor** [comment]
> LGTM, just a few comments.
> 
> Also, when running this locally with this method were you able to generate results properly? Looks like it should, just curious.

Currently running sims, seems fine so far, i produce the json files and the analyzer etc parses them and creates the *.cvs, *.md and *.json files. Excatly the same way he will do for results from raidbots :). But will wait until i have some good results to compare. Currenlty i get strange errors with simc :) need to look into them ( not related to the change of the script ).

**Review by seanpeters86 — APPROVED** [review]

**Pull request merged**